os/arch/arm/src/armv7-r/Kconfig

```diff
@@ -7,7 +7,7 @@ comment "ARMv7-R Configuration Options"
 
 config ARMV7R_HAVE_GICv2
 	bool "ARMV7R_GICv2 support"
-	default y
+	default y  if !ARCH_CHIP_BCM4390X
```

Comment:
Same query on here.
What is your intend?
1. Want to disable GICv2 always on bcm board
2. Just disable this defaultly but don't care enabling by user

If 1st is right, you should make a dependancy like below:
```
config ARMV7R_HAVE_GICv2
	bool "ARMV7R_GICv2 support"
	default y
	depends on !ARCH_CHIP_BCM4390X
....
```

Reply:
Yes.  my intention is 1st case. 

it will be changed as your guidance.
BR, HJ Lee.